featuretools/synthesis/dfs.py

```diff
@@ -159,6 +160,13 @@ def dfs(entities=None,
                 Numeric, Discrete, and Boolean. If given as
                 the string 'all', use all available variable types.
 
+        progress_callback (callable): function to be called with incremental progress updates.
```

Comment:
can we somehow include the details in the doc string that the percentages will be represented values between 0 and 100? (versus 0 to 1)